File: src/test/resources/control/for.scala
def forShow() = {
  var s = 0
  for (a <- 0 until 10) yes()
  var b = 0
  while (b + 1 < 10) {
    yes()
    b += 1
  }
  var c = 0
  while (s < 10) {
    no()
    c += 1
  }
  var d = 0
  while (d < 10) {
    no()
    d += 1
    s += 1
  }
}

File: src/test/resources/control/for.js
// different for loops, some typical, some not

function forShow() {
    var s = 0;

    for (var a = 0; a < 10; a++) yes()

    for (var b = 0; b + 1 < 10; b++) yes()

    for (var c = 0; s < 10; c++) no()

    for (var d = 0; d < 10; d++, s++) no()

    for (var i = 0, x = 2 + 3; i < x; i++) yes()

}